thirdeye-ui/src/app/components/app-bar/app-header/app-header.component.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2024 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import React, { ReactElement } from "react";
+import { useStyles } from "./app-header.styles";
+import { MenuItem, Select, Link } from "@material-ui/core";
+import { AppHeaderprops } from "./app-header.inerfaces";
+import OpenInNewIcon from "@material-ui/icons/OpenInNew";
+import { useTranslation } from "react-i18next";
+
+const Appheader = ({
```

Comment:
export const AppHeader

Reply:
export statement is at the end of the file. Since its the only export, have kept it as default.

Reply:
This is a personal preference but I like named exports because of stuff like [these](https://dev.to/phuocng/avoid-using-default-exports-a1c).